### ISSUE TITLE
AZO compression

### ISSUE THREAD

**Issue by akolman** [akolman/EggDotNet#2]
Add support for AZO compression

For any readers: please provide any examples.

**Comment by akolman** [comment]
I don't know of any C# implementation so it may need to be ported from

https://github.com/dterracino/UnEgg/tree/f21e653b02f7ab467bbc16af879e68a48dabcb2b/Source/unegg/nest/algorithm/azo

**Comment by OctopusET** [comment]
https://github.com/kippler/xunazo

**Comment by akolman** [comment]
> https://github.com/kippler/xunazo

Oh wow, I've searched for a decoder besides dterracino's and never came across that one. Thanks!

I'm curious, do you know if AZO was/is ever used outside of the .alz format? Various LLMs have not been very helpful on the topic.

**Comment by OctopusET** [comment]
I don't think so, afaik now even alzip doesn't use AZO anymore (or only for .SYS/.COM formats)

See more: https://velog.io/@soo_sudo/KVE-2018-0713-%EB%B6%84%EC%84%9D (It's in Korean)

**Comment by OctopusET** [comment]
I really hope not they or other people use for other things with this undocumented formats
